src/requirements.txt

```diff
@@ -4,3 +4,4 @@ asgiref==3.2.7
 Django==3.0.7
 pytz==2020.1
 sqlparse==0.3.1
+gunicorn==20.0.4
```

Comment:
Maybe we want to add `django-environ` here as we use it in the settings

Reply:
:+1: Good point, will do that.